Remotion/Core/Development.Analyzers/Core.Development.Analyzers.csproj

```diff
@@ -1,25 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
-    <LangVersion>9</LangVersion>
-    <Nullable>enable</Nullable>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <DevelopmentDependency>true</DevelopmentDependency>
-    <IncludeBuildOutput>false</IncludeBuildOutput>
-    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <ProjectType>Analyzer</ProjectType>
+  </PropertyGroup>
+  <Import Project="..\..\..\Build\Shared.build.props" />
+  <PropertyGroup>
     <AssemblyName>Remotion.Core.Development.Analyzers</AssemblyName>
     <AssemblyTitle>Remotion Core Analyzers</AssemblyTitle>
     <Description>Analyzer ensuring best practices when using the Remotion.Core assemblies.</Description>
   </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.8.0" />
-    <PackageReference Include="Microsoft.Net.Compilers" Version="4.1.0" />
-  </ItemGroup>
   <ItemGroup>
-    <AdditionalFiles Include="AnalyzerReleases.Shipped.md" />
-    <AdditionalFiles Include="AnalyzerReleases.Unshipped.md" />
+    <ProjectReference Remove="$(MsBuildThisFileFullPath)"/>
```

Comment:
Why do we have this line here but not in WebDevleopementAnalyzers?

Reply:
As all the projects have the core analyzers due to the reference in `Shared.build.props`, this reference needs to be removed in the core analyzer. 

Otherwise there would be a circular reference.